addons/pos_self_order/static/src/app/components/product_card/product_card.js

```diff
@@ -109,4 +111,11 @@ export class ProductCard extends Component {
             await this.selfOrder.getPricesFromServer();
         }
     }
+
+    getProductInfo() {
```

Comment:
```suggestion
    showProductInfo() {
```

seems to me that `show` is better name for this method.